src/main/java/uk/gov/companieshouse/confirmationstatementapi/model/dao/StatementOfCapitalDataDao.java

```diff
@@ -0,0 +1,28 @@
+package uk.gov.companieshouse.confirmationstatementapi.model.dao;
+
+import org.springframework.data.mongodb.core.mapping.Field;
+import uk.gov.companieshouse.confirmationstatementapi.model.tasks.TaskStatus;
+
+public class StatementOfCapitalDataDao {
+    @Field("task_status")
+    private TaskStatus taskStatus;
```

Comment:
I don't think we should reference the TaskStatus here as the task only exists in the web app and won't be relevant to a software api user. I think we agreed to derive task status from a "status' field in our previous slack conversions with Ian.

Reply:
I'm sure we mentioned having an enum for this.

Reply:
We'll just pass in a string then.

Reply:
Having the data type be an enum i think is ok.
But the Field should be called "status" not task status
And the enum should perhaps have the values of "CONFIRMED", "NO_CONFIRMED" and "RECENT_FILING"

Reply:
We seem very divided about this.